src/test/ui/consts/miri_unleashed/const_refers_to_static_cross_crate.rs

```diff
@@ -2,7 +2,7 @@
 // aux-build:static_cross_crate.rs
 #![allow(const_err)]
 
-#![feature(exclusive_range_pattern, half_open_range_patterns, const_if_match, const_panic)]
+#![feature(exclusive_range_pattern, half_open_range_patterns, const_if_match)]
```

Comment:
`const_if_match` can also be removed here

Reply:
ah no, that's a HIR check

Reply:
Yeah...